omahelsinki/static/css/colors.scss

```diff
@@ -0,0 +1,45 @@
+$hel-copper:        #00d7a7;
```

Comment:
The Helsinki theme should be taken from the `open-city-design` node package (accessible from `node_modules/open-city-design/src/scss`) after you've run `yarn install` with the latest `master` branch.